rllib/agents/a3c/a3c.py

```diff
@@ -59,6 +59,14 @@ def validate_config(config):
         logger.warning("`sample_async=True` is not supported for PyTorch! "
                        "Multithreading can lead to crashes.")
 
+    # Switch on trajectory view API by default for this algo (if None).
```

Comment:
Switch on by default for A2/3C.

Reply:
Please set this in the A2C config instead of in the code.